src/util/strencodings.cpp

```diff
@@ -587,6 +587,21 @@ bool ParseHDKeypath(const std::string& keypath_str, std::vector<uint32_t>& keypa
     return true;
 }
 
+std::string FormatKeyPath(const std::vector<uint32_t>& path)
```

Comment:
It was never the intent for `strencodings.cpp` to gain so many bitcoin-application-specific utilities, and this application is very narrow. Maybe make a new header with bip32/key specific utils?

Reply:
Moved to `bip32.{h,cpp}`